aocd/__init__.py

```diff
@@ -16,8 +16,30 @@
 from .get import get_day_and_year
 from .post import submit as _impartial_submit
 
+__all__ = [
+    "AocdError",
+    "cli",
+    "cookies",
+    "data",
+    "examples",
+    "exceptions",
+    "get_data",
+    "get",
+    "models",
+    "post",
+    "runner",
+    "submit",
+    "utils",
+]
 
-def __getattr__(name):
+data: str
+lines: list[str]
+numbers: Union[list[list[int]], list[int], int]
```

Comment:
```suggestion
lines: list[str]
numbers: Union[list[list[int]], list[int], int]
```
All the transforms were removed in v2.0.0, I decided they're out-of-scope for this project. Only `data: str` is supported.